README.md

```diff
@@ -69,207 +69,4 @@ npm publish --access public
 
 ## Useful links
 
-  * [Google Apps Script](https://developers.google.com/apps-script/)
```

Comment:
@mYokha  I've decided to remove this part. It's too complicated to maintain and keep up to date. I think that it would be easier for somebody just to discover the codebase and understand the state of the app script globals from there.

Reply:
agree

Reply:
in that case, please, adjust the 'Contribution' section of README